pkg/webui/console/components/webhook-form/index.js

```diff
@@ -60,6 +49,34 @@ const m = defineMessages({
   deleteWebhook: 'Delete Webhook',
   modalWarning:
     'Are you sure you want to delete webhook "{webhookId}"? Deleting a webhook cannot be undone!',
+  headers: 'Headers',
+  headersKeyPlaceholder: 'Authorization',
+  headersValuePlaceholder: 'Token xyz',
+  headersAdd: 'Add header entry',
+  headersValidateRequired: 'All header entry values are required. Please remove empty entries.',
+})
+
+const headerCheck = headers =>
```

Comment:
please decompose this to multiple `yup` validations

Reply:
I decided against that as it would cause the validation error object for the field to be an array of objects as well.
That would mean that we would have to either 
- pass down the error into the component in order to display it correctly. With the way we are handling form fields now, the error is shown on field level, not input component level.
- apply some logic to map the nested validation error back to an overall field error

Let me know if you have an idea to do this better.